### ISSUE TITLE
Sub-Zero stopped downloading subtitles :(

### ISSUE THREAD

**Issue by sergeantjonas** [pannal/Sub-Zero.bundle#362]
Could anyone take a look at my logs and check why I haven't downloaded any subtitles for the past few  @weeks?

[Logs_171014_21-47-11.zip](https://github.com/pannal/Sub-Zero.bundle/files/1384930/Logs_171014_21-47-11.zip)

**Comment by rb0rges** [comment]
Also here. I've tried everything but it's not automatically downloading the subtitles as before.

**Comment by pannal** [comment]
@sergeantjonas I've addressed some of the issues I've found in your logs in the current develop-2.0 branch. Nevertheless, your Plex server doesn't return information for many of your media items.

Could you do the library dance and add a clean library with your items to see if that fixes the issue?

@rb0rges you've got issues connecting to OpenSubtitles:
`ProtocolError: <ProtocolError for api.opensubtitles.org/xml-rpc: 503 Backend fetch failed>`

@opensubtitles it appeared again, today.

**Comment by opensubtitles** [comment]
Hi,

I see it. The problem is one of webservers handling API was down, so other got a lot of work. I set up script which restarts (kills process and start) the webserver which got problem. Lets see how it works in next few days, now it is ok

**Comment by sergeantjonas** [comment]
I deleted my library and re-added it, but no change I think.

**Comment by pannal** [comment]
@opensubtitles I'm receiving reports again, with `ProtocolError: <ProtocolError for api.opensubtitles.org/xml-rpc: 503 Backend fetch failed>`

**Comment by opensubtitles** [comment]
Hi, yes we are migrating data to new db server, so it is sometimes down. You can check status also here: http://api.opensubtitles.org/xml-rpc - it should work now. redis, mongodb migrated, sphinx and mariadb are on the plan...

**Comment by chethanbp521** [comment]
Same here. Force-find doesn't work too. 
[Logs_171023_19-02-21.zip](https://github.com/pannal/Sub-Zero.bundle/files/1407291/Logs_171023_19-02-21.zip)

**Comment by pannal** [comment]
@chethanbp521 your issue is unrelated. SZ is working, it just doesn't find any subtitles for your media files, because your media file names don't include enough meta information, such as format (WEB-DL, HDTV, ...), or release group. You can lower your TV scores to 330, but you'll be receiving bad subtitles from time to time.

If there's any other question, please open up a separate ticket.

**Comment by LaMpiR** [comment]
The same thing. It was working before but sometines, definately not always, but now, nothing. Would appreciate any help?



[Logs_171026_14-21-20.zip](https://github.com/pannal/Sub-Zero.bundle/files/1418301/Logs_171026_14-21-20.zip)

**Comment by pannal** [comment]
@LaMpiR yes, you've got the original problem `SSLError: ('The read operation timed out',)` on OpenSubtitles. I also experienced this yesterday.

@opensubtitles is the migration done?

**Comment by opensubtitles** [comment]
@pannal partly is done, mysql (mariadb) stays on the original server for now. I think I will be not touching config for next few weeks, so lets see how it performs. I am simulating login and SearchSubtitles() on https://www.opensubtitles.org/addons/xml_rpc_benchmark_search.php - so you can set up monitoring to see yourself if it is working or not. I am using for this quite nice opensource: http://www.phpservermonitor.org/

**Comment by LaMpiR** [comment]
So, I have the same problem, auto-download doesn't work and even when manually selecting a subtitle (search results appear), nothing happens.


[Logs_171107_14-45-14.zip](https://github.com/pannal/Sub-Zero.bundle/files/1450166/Logs_171107_14-45-14.zip)

**Comment by pannal** [comment]
You have permissions problems, @LaMpiR: `IOError: [Errno 13] Permission denied: u'/home/vuka/torrents/rtorrent/Serije/Me, MYSELF & I/Season 1/Me.Myself.and.I.S01E06.720p.HDTV.X264-DIMENSION.en.srt'`.

This is unrelated, please fix your permissions and open another ticket.

**Comment by pannal** [comment]
You have permissions problems, @LaMpiR: `IOError: [Errno 13] Permission denied: u'/home/vuka/torrents/rtorrent/Serije/Me, MYSELF & I/Season 1/Me.Myself.and.I.S01E06.720p.HDTV.X264-DIMENSION.en.srt'`.

This is unrelated, please fix your permissions and open another ticket.

**Comment by baglio** [comment]
I have this issue from september now. Yes, downloading subtitles by force-refreshing a single element does work, but SubZero stopped auto-downloading subtitles during the _periodic_ scan.
[sub-zero.zip](https://github.com/pannal/Sub-Zero.bundle/files/1544031/sub-zero.zip)

**Comment by eltismerino** [comment]
@baglio please open a separate issue. From your logs everything looks fine, though. It downloads subtitles without a problem.

**Comment by pannal** [comment]
@baglio please open a separate issue. From your logs everything looks fine, though. It downloads subtitles without a problem.

**Comment by jaimevisser** [comment]
Same happens here, timeouts and 503's

**Comment by pannal** [comment]
@jaimevisser yes, we've seen several issues with OpenSubtitles again in the last few days. Ping @opensubtitles

**Comment by opensubtitles** [comment]
hi, we are aware of this. we have some scaling issues, working on it. @pannal please contact me on my email.

**Comment by pannal** [comment]
@opensubtitles what happened to XMPP? I'm online there.
